### ISSUE TITLE
Have transparent cross-compile support

### ISSUE THREAD

**Issue by mbr0wn** [gnuradio/pybombs2#22]
I want to be able to run

    $ pybombs install gr-foo

..and `gr-foo` automatically gets cross-compiled if the current prefix has an SDK installed.

Depends on #2 and #4 (`pybombs prefix init --sdk FOO`)

**Comment by mbr0wn** [comment]
Also requires that I can set CMAKE_INSTALL_PREFIX to something other than the prefix and then use DESTDIR.

**Comment by mbr0wn** [comment]
OK, seems to work. Need to figure out how to nicely deploy, but that's another issue.
